lib/vagrant-registration/action/register.rb

```diff
@@ -42,7 +37,8 @@ def call(env)
 
           @logger.debug('Registration is skipped due to the configuration') if config.skip
 
-          @app.call(env) if Plugin.vbguest_plugin?
+          # Call next middleware in chain
+          @app.call(env)
```

Comment:
To be clear, there was indeed an issue with the initial change I made, but I addressed this in a following push. What I wanted to achieve was to consistently call the registration on the "down" hook. The existing implementation would call the registration on the "down" hook for everything except libvirt and for libvirt it would call it on the uphook. That does not make sense. You really want to make wait until the machine is reachable and then just call registration and the next middleware in the chain (aka using the downhook).

The old implementation was hooking into FolderSync for libvirt, which for some providers means the machine is already reachable, however, in the case of libvirt this is not the case. Hence, you needed to to the register on the "up" hook. I changed the code in _plugin.rb_, so that libvirt registers an action after it is known that the machine is reachable. Now, the registration itself can always occur on the "down" hook. 

Bottom line, there was initially an issue with the pull request,  but that's addressed.